roadmap/implementors-guide/src/node/backing/pov-distribution.md

```diff
@@ -4,10 +4,118 @@ This subsystem is responsible for distributing PoV blocks. For now, unified with
 
 ## Protocol
 
-Handle requests for PoV block by candidate hash and relay-parent.
+`ProtocolId`: `b"povd"`
+
+Input: [`PoVDistributionMessage`](../../types/overseer-protocol.md#pov-distribution-message)
+
+
+Output:
+
+- NetworkBridge::RegisterEventProducer(`ProtocolId`)
+- NetworkBridge::SendMessage(`[PeerId]`, `ProtocolId`, `Bytes`)
+- NetworkBridge::ReportPeer(PeerId, cost_or_benefit)
+
 
 ## Functionality
 
-Implemented as a gossip system, where `PoV`s are not accepted unless we know a `Seconded` message.
+This network protocol is responsible for distributing [`PoV`s](../../types/availability.md#proof-of-validity) by gossip. Since PoVs are heavy in practice, gossip is far from the most efficient way to distribute them. In the future, this should be replaced by a better network protocol that finds validators who have validated the block and connects to them directly.
+
+This protocol is described in terms of "us" and our peers, with the understanding that this is the procedure that any honest node will run. It has the following goals:
+  - We never have to buffer an unbounded amount of data
+  - PoVs will flow transitively across a network of honest nodes, stemming from the validators that originate them.
+
+As we are gossiping, we need to track which PoVs our peers are waiting for to avoid sending them data that they are not expecting. It is not reasonable to expect our peers to buffer unexpected PoVs, just as we will not buffer unexpected PoVs. So notification to our peers about what is being awaited is key. However it is important that the notifications system is also bounded.
```

Comment:
```suggestion
As we are gossiping, we need to track which PoVs our peers are waiting for to avoid sending them data that they are not expecting. It is not reasonable to expect our peers to buffer unexpected PoVs, just as we will not buffer unexpected PoVs. So notifying our peers about what is being awaited is key. However it is important that the notifications system is also bounded.
```

Not sure whether this was wrong in the first place.

Reply:
it was not wrong, but less clear than your suggested alternative